A3A/addons/core/Params.hpp

```diff
@@ -211,13 +218,6 @@ class Params
         texts[] = {"100", "300", "500", "1000", "1500", "2000"};
         default = 500;
     };
-    class totalVictory
-    {
-        title = $STR_A3AU_total_victory;
-        values[] = {0,1};
-        texts[] = {$STR_antistasi_dialogs_generic_button_no_text,$STR_antistasi_dialogs_generic_button_yes_text};
-        default = 0;
-    };
```

Comment:
We can't just outright remove this because I believe it'll break saves that already have it enabled

Reply:
didnt know about that

Reply:
Leave it in there but label it as deprecated or "DON'T USE", we can then remove it the update after the one that introduces this and people have had fair warning